Vizzuality/heco-invest
# feat: API for list of OpenCallApplications for private area

New API endpoint (`GET /api/v1/account/open_call_applications`) which returns list of open call applications for:
 - investor: applications for investor open calls
 - project_developer: applications for PD projects

Data returned by this endpoint are for both type of accounts same, therefore FE is in full control of what is shown (works same way as other endpoints). Endpoint supports `includes` param so it is possible to obtain details of investor/project/project_developer/open_call at any point <-- required for example for details.

Searching works slightly differently for each account type, so I have split it and different ransack query is used for Investor and PD.

## Testing instructions

New endpoint is again part of rswag documentation. Login as PD and check that you get list of applications which you have created. Login as Investor and check that you see all applications for your all open calls.

## Tracking

https://vizzuality.atlassian.net/browse/LET-890
https://vizzuality.atlassian.net/browse/LET-923
https://vizzuality.atlassian.net/browse/LET-895
https://vizzuality.atlassian.net/browse/LET-972
https://vizzuality.atlassian.net/browse/LET-924
https://vizzuality.atlassian.net/browse/LET-971
